tests/lib/rules/brace-style.js

```diff
@@ -110,19 +110,46 @@ ruleTester.run("brace-style", rule, {
         ] },
         { code: "if (a) { \nb();\n } else { \nc();\n }", options: ["allman"], errors: [
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
-            { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" }
+            { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
+            { message: CLOSE_MESSAGE_STROUSTRUP_ALLMAN, type: "BlockStatement" }
         ]},
         { code: "if (foo) {\nbaz();\n} else if (bar) {\nbaz();\n}\nelse {\nqux();\n}", options: ["allman"], errors: [
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
+            { message: CLOSE_MESSAGE_STROUSTRUP_ALLMAN, type: "IfStatement" },
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" }
         ] },
         { code: "if (foo)\n{ poop();\n} \nelse if (bar) {\nbaz();\n} else if (thing) {\nboom();\n}\nelse {\nqux();\n}", options: ["allman"], errors: [
             { message: BODY_MESSAGE, type: "ExpressionStatement" },
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
+            { message: CLOSE_MESSAGE_STROUSTRUP_ALLMAN, type: "IfStatement" },
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" },
             { message: OPEN_MESSAGE_ALLMAN, type: "IfStatement" }
         ] },
+        {
+            code: [
+                "if (foo)",
```

Comment:
I would go with the same style that is already used above (single string with "/n"). In general, we should just use multiline strings for those, I think